<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralWindow</class>
 <widget class="QWidget" name="GeneralWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RakCloud</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="mainHL">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="manager">
        <attribute name="title">
         <string>Manager</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="managerLogVL">
           <item>
            <widget class="QPlainTextEdit" name="managerLogOutput">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="managerInputHL">
             <item>
              <widget class="CommandLine" name="managerCmdInput"/>
             </item>
             <item>
              <widget class="QPushButton" name="managerCmdEnterButton">
               <property name="text">
                <string>Enter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="managerInfoAndButtonsVL">
           <item>
            <widget class="QGroupBox" name="managerSystemInfoGroup">
             <property name="maximumSize">
              <size>
               <width>800</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Sytem information</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <layout class="QFormLayout" name="managerSystemInfoFL">
                <item row="0" column="0">
                 <widget class="QLabel" name="managerVersionLabel">
                  <property name="text">
                   <string>Version:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="managerVersion">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="totalBots">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="totalBotsLabel">
                  <property name="text">
                   <string>Total bots:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="runningBotsLabel">
                  <property name="text">
                   <string>Running bots:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="runningBots">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="managerUpTimeLabel">
                  <property name="text">
                   <string>Up time:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="managerUpTime">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="managerReloadScriptsButton">
             <property name="text">
              <string>Reload manager scripts</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bots">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Bots</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListWidget" name="botList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="botLogAndInputVL">
           <item>
            <widget class="QPlainTextEdit" name="botLogOutput">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="botCmdInputAndButtonHL">
             <item>
              <widget class="CommandLine" name="botCmdInput">
               <property name="enabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="botCmdEnterButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Enter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="botInfoVL">
           <item>
            <widget class="QGroupBox" name="botInfoGroup">
             <property name="title">
              <string>System Information</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="0">
               <layout class="QFormLayout" name="botSystemInfoLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="botIdLabel">
                  <property name="text">
                   <string>Bot ID:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="botId">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="botUpTimeLabel">
                  <property name="text">
                   <string>Up time:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="botUpTime">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="proxy">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="proxyLabel">
                  <property name="text">
                   <string>Proxy:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="sampInfoGroup">
             <property name="title">
              <string>SA-MP Information</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QFormLayout" name="sampInfoFormLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="idLabel">
                  <property name="text">
                   <string>ID:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="sampId">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="nickNameLabel">
                  <property name="text">
                   <string>NickName:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="nickName">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="moneyLabel">
                  <property name="text">
                   <string>Money:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="money">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="levelLabel">
                  <property name="text">
                   <string>Level:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="level">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="healthLabel">
                  <property name="text">
                   <string>Health:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="health">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="armourLabel">
                  <property name="text">
                   <string>Armour:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLabel" name="armour">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="positionLabel">
                  <property name="text">
                   <string>Position:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLabel" name="position">
                  <property name="text">
                   <string>X: 0; Y: 0; Z: 0</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="playersLabel">
                  <property name="text">
                   <string>Players:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLabel" name="players">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="serverLabel">
                  <property name="text">
                   <string>Server:</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLabel" name="server">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botReloadScriptsButton">
             <property name="text">
              <string>Reload bot script</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandLine</class>
   <extends>QLineEdit</extends>
   <header>CommandLine.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
